blocks-config/image-gallery/class-spectra-image-gallery.php

```diff
@@ -941,6 +945,15 @@ public function render_initial_grid( $attributes ) {
 					}
 				}
 
+				// Check if the new Object Focus List is empty and the old Array Focus List is not - if so, transfer it.
+				if ( empty( $attributes['focusListObject'] ) && is_array( $attributes['focusList'] ) && 0 !== count( $attributes['focusList'] ) ) {
```

Comment:
```suggestion
				if ( empty( $attributes['focusListObject'] ) && is_array( $attributes['focusList'] ) && ! empty( $attributes['focusList'] ) ) {
```
should we do it this way

Reply:
Yes, replaced with Not Empty check.